### ISSUE TITLE
RT Failure using only -n in rt.sh still causes RT to pass.

### ISSUE THREAD

**Issue by BrianCurtis-NOAA** [ufs-community/ufs-weather-model#1416]
## Description
Running `rt.sh -n rap_control` with `feature/merra2_thompson in https://github.com/ChunxiZhang-NOAA/ufs-weather-model.git` causes comparisons to fail, but the regression test passes (is successful)

### To Reproduce (hera.intel):
```
git clone -b feature/merra2_thompson --recursive https://github.com/ChunxiZhang-NOAA/ufs-weather-model.git
cd ufs-weather-model
module purge; module reset
module use modulefiles
module load ufs_hera.intel
cd tests
./rt.sh -n rap_control
```
Log File:
```
Tue Sep 13 20:11:18 UTC 2022
Start Regression test

Compile 001 elapsed time 395 seconds. -DAPP=ATM -DCCPP_SUITES=FV3_RAP,FV3_RAP_RR
TMGP,FV3_RAP_sfcdiff,FV3_HRRR,FV3_HRRR_smoke,FV3_RRFS_v1beta,FV3_RRFS_v1nssl -D3
2BIT=ON -DMPI=ON -DCMAKE_BUILD_TYPE=Release

baseline dir = /scratch1/NCEPDEV/nems/emc.nemspara/RT/NEMSfv3gfs/develop-2022090
9/INTEL/rap_control
working dir  = /scratch1/NCEPDEV/stmp2/Brian.Curtis/FV3_RT/rt_293243/rap_control
Checking test 001 rap_control results ....
 Comparing sfcf000.nc ............ALT CHECK......NOT OK
 Comparing sfcf021.nc ............ALT CHECK......NOT OK
 Comparing sfcf024.nc ............ALT CHECK......NOT OK
 Comparing atmf000.nc ............ALT CHECK......NOT OK
 Comparing atmf021.nc ............ALT CHECK......NOT OK
 Comparing atmf024.nc ............ALT CHECK......NOT OK
 Comparing GFSFLX.GrbF00 .........NOT OK
 Comparing GFSFLX.GrbF21 .........NOT OK
 Comparing GFSFLX.GrbF24 .........NOT OK
 Comparing GFSPRS.GrbF00 .........NOT OK
 Comparing GFSPRS.GrbF21 .........NOT OK
 Comparing GFSPRS.GrbF24 .........NOT OK
 Comparing RESTART/coupler.res .........OK
 Comparing RESTART/fv_core.res.nc .........OK
 Comparing RESTART/fv_core.res.tile1.nc ............ALT CHECK......NOT OK
 Comparing RESTART/fv_core.res.tile2.nc ............ALT CHECK......NOT OK
 Comparing RESTART/fv_core.res.tile3.nc ............ALT CHECK......NOT OK
 Comparing RESTART/fv_core.res.tile4.nc ............ALT CHECK......NOT OK
 Comparing RESTART/fv_core.res.tile5.nc ............ALT CHECK......NOT OK
 Comparing RESTART/fv_core.res.tile6.nc ............ALT CHECK......NOT OK
 Comparing RESTART/fv_srf_wnd.res.tile1.nc ............ALT CHECK......NOT OK
 Comparing RESTART/fv_srf_wnd.res.tile2.nc ............ALT CHECK......NOT OK
 Comparing RESTART/fv_srf_wnd.res.tile3.nc ............ALT CHECK......NOT OK
 Comparing RESTART/fv_srf_wnd.res.tile4.nc ............ALT CHECK......NOT OK
 Comparing RESTART/fv_srf_wnd.res.tile5.nc ............ALT CHECK......NOT OK
 Comparing RESTART/fv_srf_wnd.res.tile6.nc ............ALT CHECK......NOT OK
 Comparing RESTART/fv_tracer.res.tile1.nc ............ALT CHECK......NOT OK
 Comparing RESTART/fv_tracer.res.tile2.nc ............ALT CHECK......NOT OK
 Comparing RESTART/fv_tracer.res.tile3.nc ............ALT CHECK......NOT OK
 Comparing RESTART/fv_tracer.res.tile4.nc ............ALT CHECK......NOT OK
 Comparing RESTART/fv_tracer.res.tile5.nc ............ALT CHECK......NOT OK
 Comparing RESTART/fv_tracer.res.tile6.nc ............ALT CHECK......NOT OK
 Comparing RESTART/phy_data.tile1.nc ............ALT CHECK......NOT OK
 Comparing RESTART/phy_data.tile2.nc ............ALT CHECK......NOT OK
 Comparing RESTART/phy_data.tile3.nc ............ALT CHECK......NOT OK
 Comparing RESTART/phy_data.tile4.nc ............ALT CHECK......NOT OK
 Comparing RESTART/phy_data.tile5.nc ............ALT CHECK......NOT OK
 Comparing RESTART/phy_data.tile6.nc ............ALT CHECK......NOT OK
 Comparing RESTART/sfc_data.tile1.nc ............ALT CHECK......NOT OK
 Comparing RESTART/sfc_data.tile2.nc ............ALT CHECK......NOT OK
 Comparing RESTART/sfc_data.tile3.nc ............ALT CHECK......NOT OK
 Comparing RESTART/sfc_data.tile4.nc ............ALT CHECK......NOT OK
 Comparing RESTART/sfc_data.tile5.nc ............ALT CHECK......NOT OK
 Comparing RESTART/sfc_data.tile6.nc ............ALT CHECK......NOT OK

  0: The total amount of wall time                        = 440.733187
  0: The maximum resident set size (KB)                   = 1050028

Test 001 rap_control FAIL


REGRESSION TEST WAS SUCCESSFUL
Tue Sep 13 20:31:54 UTC 2022
Elapsed time: 00h:20m:37s. Have a nice day!
```

**Comment by DusanJovic-NOAA** [comment]
In run_test.sh this must be removed:

https://github.com/ufs-community/ufs-weather-model/blob/8925cdb03aec5324e4bd256253a61992ea2101d7/tests/run_test.sh#L376

**Comment by BrianCurtis-NOAA** [comment]
Fixed in #1554
